config/logger.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 const (
-	LogRotateSize           = 10000
+	LogRotateSize           = 100
```

Comment:
```suggestion
	LogRotateSizeMB           = 100
```

Should we add the unit as a suffix like we do for other similar constants (e.g. time related)? It may help to prevent this from happening again.